_test/select14.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+type T struct {
+	c1 chan string
+	c2 chan string
+}
+
+func main() {
+	t := &T{}
+	t.c2 = make(chan string)
+
+	go func(c chan string) { c <- "done" }(t.c2)
+
+	select {
+	case msg := <-t.c1:
+		println("received from c1:", msg)
```

Comment:
Is this case useful for the test?

Reply:
It makes sense to have multiple comm clauses to test for correct branching, at CFG level and at run level.